yarn.lock

```diff
@@ -286,12 +292,16 @@
   resolved "https://registry.yarnpkg.com/@types/mocha/-/mocha-2.2.48.tgz#3523b126a0b049482e1c3c11877460f76622ffab"
 
 "@types/node@*":
-  version "10.0.0"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.0.0.tgz#c40f8e07dce607d3ef25a626b93a6a7cdcf97881"
+  version "10.3.6"
```

Comment:
we should make sure that we use typings which corresponding to our node version, not latest

Reply:
how to force it?

that comes from a dependency.